### PR TITLE
Prepare deployments scripts for testnet deployment

### PR DESCRIPTION
**Pull request by fbielejec**
- configurable Oracle owner account
- explicit owners in config
- Migrations on Azero
- 0 deploys Migrations, every script updates it
- checks if previous migrations has been completed

### CODE REVIEW

File: azero/contracts/migrations/lib.rs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        #[ink(constructor)]
+        pub fn new(owner: AccountId) -> Self {
+            Self {
+                last_completed_migration: 0,
+                owner,
+            }
+        }
+
+        #[ink(message)]
+        pub fn set_completed(&mut self, completed: u32) {
```
Comment (JanKuczma) [inline azero/contracts/migrations/lib.rs:40]: I think there is an "is_owner" check missing

File: azero/contracts/migrations/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+edition = "2021"
+
+[dependencies]
+ink = { version = "=4.3.0", default-features = false }
+
+scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
+scale-info = { version = "2.6", default-features = false, features = ["derive"], optional = true }
+
+[dev-dependencies]
+ink_e2e = "4.2.0"
```
Comment (JanKuczma) [inline azero/contracts/migrations/Cargo.toml:14]:
```suggestion
```
No need for it, right?

File: azero/contracts/migrations/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+[package]
+name = "migrations"
+version = "0.1.0"
+authors = ["[your_name] <[your_email]>"]
```
Comment (JanKuczma) [inline azero/contracts/migrations/Cargo.toml:4]:
```suggestion
authors = ["Cardinal Cryptography"]
```

**Review by JanKuczma — CHANGES_REQUESTED** [review]

File: azero/contracts/migrations/Cargo.toml (hunk 2 above)
Comment (fbielejec) [reply azero/contracts/migrations/Cargo.toml]: indeed, removed

File: azero/contracts/migrations/lib.rs (hunk 1 above)
Comment (fbielejec) [reply azero/contracts/migrations/lib.rs]: oh yeah, very good catch!

**Pull request merged**